examples/training_scripts/nashpobench/nashpobench.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
OK, you probably need this for comparisons, right? Say, to verify the simulator back-end by comparing against local back-end with sleeping.

At some point, we should probably do this with a single wrapper (as David suggested), so we don't have to do that for every blackbox.

Note that the simulator back-end does not need such a script at all by now, because it uses the blackbox directly.

Reply:
I agree but it's probably better to do this in a separate PR

Reply:
Sure.